### DISCUSSION TITLE
Denied for GitHub education because "The image you selected does not appear to contain your school name."

### DISCUSSION THREAD

**Discussion by stautonico** [community/community#66733] [GitHub Education]
### Hi everyone,

Hello. I'm trying to re-validate my academic status, but the automated service is giving me a hard time. Every single one of my schools documents uses the abbreviation instead of the full name, which the automated service does not detect. Even checks made out to the school is supposed to be addressed to the abbreviation instead of the full name. What am I supposed to do in this situation?

**Comment by Mythologyli** [comment]
I think you can create a support ticket.
https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket

**Reply by stautonico** [reply]
Unfortunately not since they said that they no longer do manual review for education and it prevents me from making a ticket, but I'll try without selecting a topic.
